hydrosdk/modelversion.py

```diff
@@ -492,15 +500,15 @@ def to_proto(self):
             image_sha=self.image.sha256
         )
 
-    def __init__(self, id, name, version, contract, runtime, image, cluster, status, metadata=None,
-                 install_command=None):
+    def __init__(self, id, name, version, contract, cluster: Cluster, runtime=None, image=None,
+                 status=None, metadata=None, install_command=None):
```

Comment:
☝️  I think that typehints should be more consistent. Especially when there is single expected type for a parameter.